tox.ini

```diff
@@ -1,32 +1,27 @@
 [tox]
 minversion = 3.10
 envlist = py310,py39,py38,py37,lint
-skipsdist = True
 
 [testenv]
 usedevelop = True
-install_command = pip install -U {opts} {packages}
```

Comment:
By removing this, we lose the `-U` part which tells pip to first upgrade itself to the newest version.

I don't see a particular reason we'd need to do that.